sources/main.cpp

```diff
@@ -120,6 +126,8 @@ main(int argc, char* argv[]) {
     LOG(info) << " ============================================================ " << LOG_ENDL;
     LOG(info) << MESS(DFLEnded, context.basename(), timerGlobal.elapsed()) << LOG_ENDL;
 
+    xmlCleanupParser();
```

Comment:
Factorize

Reply:
by that you mean create a method?

Reply:
Yes, a method that does both lines, since it is used in multiple location